### ISSUE TITLE
teamviewer/wargods

### ISSUE THREAD

**Issue by deNchj** [3a1/Evelion#1]
is this trick able to bypass wargods and find a person connecting to my computer on teamviewer

**Comment by 3a1** [comment]
To bypass a wargods you need just to change signature of the compiled file, but as i know every new compilation signature is changing, so i think to bypass wargods you need just to compile it by yourself and that all. I dont get what you mean by a person connecting to my computer on teamviewer. If you mean that someone will check your pc by teamviewer , that is your own business to hide it, cheat is only .exe file that not injecting anything into cs process, so you can maybe start it from usb drive, and when someone will check you pc just unplug usb drive? ;)

**Comment by deNchj** [comment]
yes, I'm talking about when there is teamwiver control. But I don't understand this aspect of how to bypass through wargods
it would be very good if you could make a cheat that bypasses wargods and hides in a normal way and the content of the cheat you don't need to add aimbot it would be very good if you add something like hitbox fix and for example a dll where bullets can go well. there are many programs available in usb that we can see that when we unplug and plug it in, it does flash plug and unplug :) :)
